client/extensions/woocommerce/app/order/order-payment/table.js

```diff
@@ -257,6 +263,7 @@ class OrderRefundTable extends Component {
 						isEditable
 						currency={ order.currency }
 						label={ translate( 'Shipping' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **20** times:
`translate( 'Shipping address' )` *ES Score: 9*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/store-price-input-max-button)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).